From 08817df0799e7c378b652c51d242d41a21d99b13 Mon Sep 17 00:00:00 2001
From: "tristan.ramseyer" <tristan.ramseyer@stromerbike.com>
Date: Tue, 3 Sep 2024 14:16:05 +0200
Subject: [PATCH] REVERT QTBUG 77006

ensure all children of a widget get updated when a stylesheet-changes

Upstream-Status: Inappropriate [stromer-specific]
Signed-off-by: tristan.ramseyer <tristan.ramseyer@stromerbike.com>
---
 src/widgets/styles/qstylesheetstyle.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/widgets/styles/qstylesheetstyle.cpp b/src/widgets/styles/qstylesheetstyle.cpp
index 9fcb8ba522..e7f13e33fa 100644
--- a/src/widgets/styles/qstylesheetstyle.cpp
+++ b/src/widgets/styles/qstylesheetstyle.cpp
@@ -2767,11 +2767,6 @@ static void updateObjects(const QList<const QObject *>& objects)
         if (auto widget = qobject_cast<QWidget*>(const_cast<QObject*>(object))) {
             widget->style()->polish(widget);
             QCoreApplication::sendEvent(widget, &event);
-            QList<const QObject *> children;
-            children.reserve(widget->children().size() + 1);
-            for (auto child: qAsConst(widget->children()))
-                children.append(child);
-            updateObjects(children);
         }
     }
 }
-- 
2.34.1

